### ISSUE TITLE
Wrong item, false data

### ISSUE THREAD

**Issue by grey1989** [shuoros/cifar100-resnet50-pytorch#2]
Use the training set to train, and the test is also tested on the training set. Are kidding me？
'train': torchvision.datasets.CIFAR100(root='data', train=True, download=True, transform=data_transforms['train']),
    'test': torchvision.datasets.CIFAR100(root='data', **_train=False*_*,download=True, transform=data_transforms['test']   
The above test set defaults to True

**Comment by shuoros** [comment]
Hello @grey1989,

Thank you for your attention. This was a university course project, and I did not really pay attention to this problem, and now that I see it, I laugh at myself =)). I'll fix the issue and re-train the model. Although I'm pretty sure I will not get these results again.
